_includes/sidebar.html

```diff
@@ -40,17 +40,16 @@
       <li><a href="https://codelabs.developers.google.com/codelabs/flutter-firebase/index.html#0">Firebase for Flutter - Codelab</a></li>
     </ul>
 
-
-  <li class="sidebar-title">Test, deploy, workflows</li>
+  <li class="sidebar-title">Development and tools</li>
 
     <ul class="sidebar-items">
+      <li><a href="/intellij-ide/">Using IntelliJ</a></li>
```

Comment:
Actually, now this also applies to Android Studio? Maybe reword to something like 'Flutter IDE support'

Reply:
Makes sense - updated to `Using Flutter IDEs`